src/librustc_span/symbol.rs

```diff
@@ -837,6 +837,7 @@ symbols! {
         warn,
         wasm_import_module,
         wasm_target_feature,
+        wasm_nontrapping_fptoint: "nontrapping-fptoint",
```

Comment:
FWIW the "wasm" prefix on the ones above are because the symbol literally has `wasm_*` in the name (e.g. `#[link(wasm_import_module = "...")]`, so this may be best called `nontraping_fptoint` perhaps?